### ISSUE TITLE
Inconsistent errors in the editor compared to eslint output

### ISSUE THREAD

**Issue by sylv3r** [Microsoft/vscode-eslint#552]
My editor is pointing out a lot of issues tagged as `[eslint]` while running eslint in the command line does not report any issue.

As an example : 
`[eslint] 'React' is defined but never used. Allowed unused vars must match /^_/. (no-unused-vars)`
`[eslint] 'ThemeProvider' is defined but never used. Allowed unused vars must match /^_/. (no-unused-vars)`

While (I voluntarily explicitly called here the config file just to prove the point, but the result is the same without it) : 
```
$ yarn eslint -c .eslintrc.js src
yarn run v1.9.4
$ [my absolute path]/node_modules/.bin/eslint -c .eslintrc.js src
✨  Done in 2.29s.
```

My config is in `.js` format `.eslintrc.js`

Here's my eslint config : 
```
module.exports = {
  extends: 'airbnb-base',
  env: {
    node: true,
    es6: true,
  },
  parser: 'babel-eslint',
  plugins: [
    'jsdoc',
    'react',
    'babel',
    'import',
  ],
  rules: {
    semi: 'off',
    'space-before-function-paren': ['error', 'always'],
    'no-unused-vars': ['error', { argsIgnorePattern: '^_' }],
    'arrow-parens': ['error', 'as-needed', { requireForBlockBody: false }],
    'object-curly-newline': ['error', { 'consistent': true }],
    'no-unused-expressions': ['off', { allowShortCircuit: true }],
    'no-unused-vars': ['error', {
      argsIgnorePattern: '^_',
      varsIgnorePattern: '^_',
    }],
  },
  parserOptions: {
    ecmaFeatures: {
      jsx: true
    }
  },
  settings: {
    'import/resolver': {
      node: {
        extensions: ['.js', '.jsx', '.mjs'],
      },
    },
  },
}

```

**Comment by dbaeumer** [comment]
It would be helpful if you could provide a GitHub repository I can clone that demos this.

Looking on how you call this I suspect that you need to set the `src` folder as a working directory. See `eslint.workingDirectories` [here](https://github.com/Microsoft/vscode-eslint)
